def print_error(err):
    print(err)
    sys.exit(1)
